plinth/modules/users/tests/test_actions.py

```diff
@@ -100,6 +96,8 @@ def tearDown(self):
         for group in self.groups:
             self.delete_group(group)
 
+        security.set_restricted_access(True)
```

Comment:
Rather than enabling restricted access when the test is done, shouldn't the previous setting be restored?